src/diffTool/diffProcessor.ts

```diff
@@ -29,7 +32,11 @@ export async function diffRaml(
   ramlToolLogger.info(
     `Finding differences between flattened JSON-LD of ${leftRaml} and ${rightRaml}`
   );
-  return findJsonDiffs(leftGraph, rightGraph);
+  const diffs = findJsonDiffs(leftGraph, rightGraph);
+  if (rules != null) {
```

Comment:
Do we need to check for non empty rules before applying?

Reply:
Its done in applyRules function

Reply:
Then I think we can do the null check in applyRules function as well

Reply:
null check is also done in applyRules, this is an additional check to avoid invoking if user do not want to apply rules

Reply:
null check is also done in applyRules. I will remove it here